applications/MappingApplication/tests/test_nearest_neighbor_mapper_iga.py

```diff
@@ -0,0 +1,325 @@
+
+        cls.current_model = KM.Model()
+        cls.model_part_origin = cls.current_model.CreateModelPart("origin")
+        cls.model_part_destination = cls.current_model.CreateModelPart("destination")
+
+        # list of variables involved in the Mapper-Tests
+        cls.model_part_origin.AddNodalSolutionStepVariable(KM.PRESSURE)
+        cls.model_part_origin.AddNodalSolutionStepVariable(KM.FORCE)
+        cls.model_part_origin.AddNodalSolutionStepVariable(KM.DISPLACEMENT)
+
+        cls.model_part_destination.AddNodalSolutionStepVariable(KM.TEMPERATURE)
+        cls.model_part_destination.AddNodalSolutionStepVariable(KM.VELOCITY)
+        cls.model_part_destination.AddNodalSolutionStepVariable(KM.REACTION)
+        cls.model_part_destination.AddNodalSolutionStepVariable(KM.MESH_DISPLACEMENT)
+
+        cls.model_part_origin.ProcessInfo[KM.DOMAIN_SIZE] = 3 # needed for the partitioner!
+        cls.model_part_destination.ProcessInfo[KM.DOMAIN_SIZE] = 3 # needed for the partitioner!
+        cls.model_part_origin.ProcessInfo[KM.TIME] = 0.0 # needed for the check-processes
+        cls.model_part_destination.ProcessInfo[KM.TIME] = 0.0 # needed for the check-processes
+        cls.model_part_origin.ProcessInfo[KM.DELTA_TIME] = 1.0 # needed for the check-processes
+        cls.model_part_destination.ProcessInfo[KM.DELTA_TIME] = 1.0 # needed for the check-processes
+
+        cls.ReadModelParts()
+
+    @classmethod
+    def ReadModelParts(cls):
+        # Read the origin input and create the elements and conditions
+        KM.CadJsonInput(cls.input_file_origin).ReadModelPart(cls.model_part_origin)
+        origin_interface_brep_surface = cls.model_part_origin.GetGeometry(2)
+        origin_interface_quadrature_point_geometries = KM.GeometriesVector()
+        origin_interface_brep_surface.CreateQuadraturePointGeometries(origin_interface_quadrature_point_geometries, 3)
+        origin_interface_sub_model_part = cls.model_part_origin.CreateSubModelPart("neumann_boundary_iga")
+
+        # Create properties for the elements
+        if not cls.model_part_origin.HasProperties(1):
+            cls.model_part_origin.CreateNewProperties(1)
+
+        shell_properties = cls.model_part_origin.GetProperties()[1]
+        condition_id = 1
+        for i in range(0, len(origin_interface_quadrature_point_geometries)):
+            origin_interface_sub_model_part.CreateNewCondition('LoadCondition', condition_id, origin_interface_quadrature_point_geometries[i], shell_properties)
+            condition_id += 1
+
+        for node in cls.model_part_origin.Nodes:
+            origin_interface_sub_model_part.AddNode(node)
+
+        # Read the destination model part 
+        KM.ModelPartIO(cls.input_file_destination).ReadModelPart(cls.model_part_destination)
+
+    # No inverse mapping for this mapper is defined as the origin must always be IGA
+    def test_InverseMap_constant_scalar(self):
```

Comment:
do you need to implement them bcs they are overriding the behavior in the baseclass?

If so then please add some `skipIf` to at least log that they are not implemented